src/components/Quotes.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './css/quotes.css';
+
+const Quotes = () => {
+  const [quotes, setQuotes] = React.useState([]);
+  const [randomQuote, setRandomQuote] = React.useState('');
+
+  React.useEffect(() => {
+    async function fetchData() {
+      const response = await fetch('https://type.fit/api/quotes');
+      const data = await response.json();
+
+      setQuotes(data);
+      const randIndex = Math.floor(Math.random() * data.length);
+      setRandomQuote(data[randIndex]);
+    }
+    fetchData();
+  }, []);
+
+  const getNewQuote = () => {
+    const randIndex = Math.floor(Math.random() * quotes.length);
+    setRandomQuote(quotes[randIndex]);
+  };
+
+  return (
+    <div style={{ backgroundColor: 'transparent', minHeight: '100vh' }}>
+      <div className="jumbotron">
+        <div className="card">
+          <div className="card-body">
+            {randomQuote ? (
+              <>
+                <p className="card-text">
+                  &quot;
+                  {randomQuote.text}
+                  &quot;
+                </p>
+                <span className="card-title">
+                  - Author:
+                  {' '}
+                  {randomQuote.author || 'No Author'}
+                </span>
+              </>
+            ) : (
+              <h2>Loading</h2>
+            )}
+          </div>
```

Comment:
- [ ] [optional] @ArnaudBand Great work, it has been nice ensuring all quotes are aligned the same side, currently some seems to be aligned to the left and others to the center